routes/index.js

```diff
@@ -14,12 +16,37 @@ router.get("/login", function (req, res, next) {
   res.render("login");
 });
 
-router.get('/signup/phone', function (req, res, next) {
-  res.render('signup-phone', { title: '배민 - 전화번호 인증'})
+router.get("/signup/phone", function (req, res, next) {
+  res.render("signup-phone", { title: "배민 - 전화번호 인증" });
 });
 
 router.get("/signup/detail", function (req, res, next) {
+  const signupError = req.session?.signupError;
+  if (signupError) {
+    delete req.session.signupError;
+    res.render("signup-detail", { error: signupError });
+    return;
+  }
+
   res.render("signup-detail");
 });
 
+router.post("/signup", async function (req, res, next) {
+  const { email, password, nickname } = req.body;
+
+  const foundUser = await db.find({ email });
+
+  if (foundUser.length) {
```

Comment:
```suggestion
  if (foundUser) {
```

**알흠다운** 코드를 위해서는 이것이 맞다고 봅니다. 어차피 zero or one 이기 때문에 빈 배열이라면 [] == false 취급 되겠지요.

Reply:
https://github.com/woowa-techcamp-2021/baemin-17/pull/62#discussion_r666334857
참고 바랍니다~

Reply:
오읭? 빈 배열은 `truthy` 하지 않나영?

Reply:
아아아 복수가 아닌 데이터형에 .length를 쓰는게 좋지 않지만 그렇게 하죠!